docs/docs/guides/multi-stage-application-delivery.md

```diff
@@ -0,0 +1,340 @@
+    the application.
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/application-dev.yaml" %}
+    ```
+
+=== "dev/Chart.yaml"
+
+    The `Chart.yaml` that is required by helm:
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/chart.yaml" %}
+    ```
+
+=== "dev/values.yaml"
+
+    The `values.yaml` file contains configurable properties
+    for the application, such as the version of the service that should be deployed,
+    and other metadata that is passed to the `KeptnAppContext`, such as the Git
+    commit ID that caused the deployment of a new version (this value for the
+    `commitID` property is set automatically using the `$ARGOCD_APP_REVISION`
+    environment variable provided by Argo CD).
+    During the promotion phase of a new deployment, a pull request to
+    copy the content of this file to the helm chart containing the helm chart for
+    the `production` stage will be created.
+    Therefore, after merging the pull request, the version that has been deployed into
+    `dev` and has passed all checks there, will then be deployed into `production`.
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/values-dev.yaml" %}
+    ```
+
+    1. The commitID will be automatically set by Argo CD
+
+**Production Stage:**
+
+=== "production/templates/application.yaml"
+
+    This file contains the manifests for the application in the `dev` stage,
+    including the Kubernetes `Deployment`/`Service` definition,
+    as well as the `KeptnAppContext` that contains
+    a `metadata` section which includes properties that should be added
+    to the OpenTelemetry spans that are generated during the deployment of
+    the application.
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/application-production.yaml" %}
+    ```
+
+=== "production/Chart.yaml"
+
+    The `Chart.yaml` that is required by helm:
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/chart.yaml" %}
+    ```
+
+=== "production/values.yaml"
+
+    The `values.yaml` file contains configurable properties
+    for the application, such as the version of the service that should be deployed,
+    and other metadata that is passed to the `KeptnAppContext`, such as the Git
+    commit ID that caused the deployment of a new version (this value for the
+    `commitID` property is set automatically using the `$ARGOCD_APP_REVISION`
+    environment variable provided by Argo CD).
+    The content of this file is modified by the pull requests created in the
+    promotion phase of the deployment in `dev` in order to promote a new
+    service version into `production`.
+
+    ```yaml
+    {% include "./assets/multi-stage-delivery/values-production.yaml" %}
+    ```
+
+    1. The commitID will be automatically set by Argo CD
+    2.  The traceParent will be automatically set by the GitHub action triggered by the promotion task
+
+<!-- markdownlint-enable MD046 -->
+
+After creating all the files listed above, do not forget to push
```

Comment:
this bit is difficult to understand if it is part of the tab or not in the website view, I would suggest adding a subtitle

Reply:
![image](https://github.com/keptn/lifecycle-toolkit/assets/89971034/c609293e-2375-45e7-a1a7-2e791a998656)
<img width="690" alt="image" src="https://github.com/keptn/lifecycle-toolkit/assets/89971034/c1dc33e8-cfe8-40ff-9dc0-fb07ab51d3ab">